### ISSUE TITLE
[Statistics][Enhancement] Artist Stats Page

### ISSUE THREAD

**Issue by Satanarious** [harbassan/spicetify-apps#78]
Like the Playlist stats page, an Artist Stats page would be great. This would include:
- Number of times artist appeared in tracks
- Hours/Minutes listened to
- Number of times the artist appeared in individual user playlists
- The artist's global ranking is on Spotify or Last.fm
And much more.....

Regarding what the Last.fm or Spotify API allows, Is this possible?

**Comment by harbassan** [comment]
Anything to do with counting tracks is possible on my end, so the number of appearances in your library etc. Anything else just depends on the API, most likely last.fm because spotify's is pretty limited. I'll recheck what they provide though and see what I can do.

**Comment by harbassan** [comment]
Last.fm artists info, not that much here tbh:


![Image](https://github.com/harbassan/spicetify-apps/assets/84175605/fd7f2174-c6a7-4ce3-abdf-072dff3e4d4c)

**Comment by harbassan** [comment]
feel free to look around the apis though for this and the other ticket, and if you find something interesting I could add it

**Comment by harbassan** [comment]
yeah thats what i was looking for, but there doesnt seem to be anything like that.

**Comment by Satanarious** [comment]
What about parsing the user profile webpage? Because that data is available for everyone to see.

**Comment by harbassan** [comment]
Ive never actually done web crawling/parsing so I dont know whether thats an option.

**Comment by Satanarious** [comment]
I think it should be easier that way.
